src/Payload/Options.php

```diff
@@ -20,6 +20,7 @@ public function __construct(
         public readonly ?bool $forceAccount = null,
         public readonly ?ReturnUrl $returnUrl = null,
         public readonly ?array $signers = null,
+        public readonly ?bool $pathfinding = null,
```

Comment:
@WietseWind what does this default to on the API side? False? I'm assuming we can leave it out of the payload?

Reply:
It can be left out, which then defaults to `false` on our end.